code/mob/living/silicon.dm

```diff
@@ -238,6 +237,11 @@
 	else
 		return ..(message)
 
+/mob/living/silicon/var/regex/monospace_say_regex = new(@"`([^`]+)`", "g")
```

Comment:
strictly pathed vars are kinda blegh

Reply:
Not sure what you mean? Would the alternative be just creating these in the proc itself?

Reply:
```
/mob/living/silicon
    var/regex/monospace_say_regex = regex(@"`([^`]+)`", "g")
```
relative pathing
```
/mob/living/silicon/var/regex/monospace_say_regex = regex(@"`([^`]+)`", "g")
```
strict pathing

Reply:
they're not really blegh for constant ones like this, also it should be `const`